packages/smart-contracts/src/contracts/ERC20TransferableReceivable.sol

```diff
@@ -158,15 +158,16 @@ contract ERC20TransferableReceivable is ERC721, ERC721Enumerable, ERC721URIStora
    * @param paymentReference A reference for the payment.
```

Comment:
Comment:

On [line 143 - 152](https://github.com/RequestNetwork/requestNetwork/pull/1097/files#diff-58d3632849e76bbd7f443b5978ff1800d7e3bbf7bd87b13d8152b238107229c2R143-R152) the `TransferableReceivablePayment` is emitted. Are all of these Event data fields absolutely necessary?

Reply:
We can remove paymentProxy, tokenAddress, and paymentId here to save some gas.